.vscode/setup.sh

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
+
+#Activate Python
+python -i
+
+#Install PIP
+python -m pip --version
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+
+#Install Poetry
+echo 'poetry --version' poetry --version
+
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+
+poetry install
+
+echo 'poetry --version' poetry --version
+
+#Clone the repository
+git clone https://github.com/sunnygunjesh/DevOps-Course-Starter.git
+
+#Setup environment parameters
+cp .env.template .env 
```

Comment:
This is useful for running it locally, but is not how you would set up your servers. You would need a secure way of getting environment variables onto the server, rather than expecting them to be included alongside the code.

Environment variables are meant to vary between environments, and can store sensitive values.

Reply:
Have re-written setup.sh, could you please review https://github.com/sunnygunjesh/DevOps-Course-Starter/blob/Module_4/setup.sh ?